Author: Guillem Jover <guillem@debian.org>
Description: Fix various typos
Warned-by: lintian
Origin: vendor
Forwarded: yes
Applied-Upstream: 2.5.1+, commit:a56993dfa3b21c53afac3e79066d5094bbf12371

---
 man/man3/attr_multi.3  |    6 +++---
 man/man3/attr_remove.3 |    2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/man/man3/attr_multi.3
+++ b/man/man3/attr_multi.3
@@ -137,7 +137,7 @@ field contains the actual size of the at
 The
 .I am_flags
 field may be set to the \f4ATTR_ROOT\fP flag.
-If the process has appropriate priviledges,
+If the process has appropriate privileges,
 the ROOT namespace will be searched for the named attribute,
 otherwise the USER namespace will be searched.
 .TP
@@ -151,7 +151,7 @@ fields contain the new value for the giv
 The \f4ATTR_ROOT\fP flag may be set in the
 .I am_flags
 field.
-If the process has appropriate priviledges,
+If the process has appropriate privileges,
 the ROOT namespace will be searched for the named attribute,
 otherwise the USER namespace will be searched.
 The \f4ATTR_CREATE\fP and the \f4ATTR_REPLACE\fP flags
@@ -181,7 +181,7 @@ fields are not used and are ignored.
 The
 .I am_flags
 field may be set to the \f4ATTR_ROOT\fP flag.
-If the process has appropriate priviledges,
+If the process has appropriate privileges,
 the ROOT namespace will be searched for the named attribute,
 otherwise the USER namespace will be searched.
 .PP
--- a/man/man3/attr_remove.3
+++ b/man/man3/attr_remove.3
@@ -48,7 +48,7 @@ refers to the file descriptor associated
 If the attribute
 .I attrname
 exists, the attribute name and value will be removed from the
-fileystem object.
+filesystem object.
 The
 .I flags
 argument can contain the following symbols bitwise OR\'ed together:
